src/FSharp.Editing.VisualStudio.v2015/FSharp.Editing.VisualStudio.v2015.fsproj

```diff
@@ -255,4 +211,212 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.5' Or $(TargetFrameworkVersion) == 'v4.5.1' Or $(TargetFrameworkVersion) == 'v4.5.2' Or $(TargetFrameworkVersion) == 'v4.5.3' Or $(TargetFrameworkVersion) == 'v4.6' Or $(TargetFrameworkVersion) == 'v4.6.1' Or $(TargetFrameworkVersion) == 'v4.6.2' Or $(TargetFrameworkVersion) == 'v4.6.3')">
+      <ItemGroup>
+        <Reference Include="Microsoft.VisualStudio.CoreUtility">
```

Comment:
with the hintpath (and the file being present obviously) is there any chance msbuild would go somewhere else?

With the paket.lock file we have certitude of each package version, if we'd need several versions of one given nuget we'd put those in a different group (in paket parlance those are named set of direct dependencies) and they'll endup in different directories.

Here the group is `vssdk` and the file in hintpath is from this nuget package: `Microsoft.VisualStudio.CoreUtility (14.3.25407)`